CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ All notable changes to this project will be documented in this file. If a contri
 
 ## Unreleased
 
-- n/a
+- Add support for SafeAreaView when using styled-components in a React Native project
```

Comment:
Can you add a small note linking to this PR please? :) `(see [#1339](https://github.com/styled-components/styled-components/pull/1339))`